GVFS/GVFS.Common/NuGetUpgrader/InstallManifestPlatform.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace GVFS.Common.NuGetUpgrader
+{
+    public class InstallManifestPlatform

```

Comment:
Naming nit. Couple points. 1. The class is named InstallManifestPlatform, but at many places where it is used, the instance is named platformInstallManifest. This makes it look a little twisted - is this a platform or a manifest. 2. The word platform is also bit loaded with Mac and Windows specific code grouped into GVFSPlatform class, would be nice if we could find an alternate name. Also the class is used to hold a list of Manifest entries so it looks neither like a platform nor a manifest but a data source for manifests.

One more question is do we need a container class, just to hold a list of manifest entries? I mean does having a separate class help in parsing, mocking or UT?